schema_cache.rst

```diff
@@ -0,0 +1,131 @@
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`Upserts <upsert>`                    | Primary keys                                                                  |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`insert_update`                       | Primary keys (in order to return the Location header)                         |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`OPTIONS requests <options_requests>` | View primary key columns and INSTEAD OF TRIGGERS                              |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`open-api`                            | Table columns, primary keys and foreign keys                                  |
++                                            +-------------------------------------------------------------------------------+
+|                                            | View columns and INSTEAD OF TRIGGERS                                          |
++                                            +-------------------------------------------------------------------------------+
+|                                            | Function's metadata                                                           |
++--------------------------------------------+-------------------------------------------------------------------------------+
+
+The Stale Schema Cache
+----------------------
+
+When you make changes on the metadata mentioned above while PostgREST is running, the schema cache turns stale. Future requests you make related to these changes will need the :ref:`schema to be reloaded <schema_reloading>`; otherwise, you'll get an error instead of the expected result.
+
+For instance, let's see what would happen if you have a stale schema for foreign key relationships and function metadata:
+
+Stale Foreign Key Relationships
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Suppose you add a ``cities`` table to your database. This table has a foreign key ``country_id`` referencing an existing ``countries`` table. Then, you make a request to get the data from ``cities`` and their ``countries``:
+
+.. code-block:: http
+
+  GET /cities?select=name,country:countries(id,name) HTTP/1.1
+
+But instead, you get an error message that looks like this:
+
+.. code-block:: json
+
+  {
+    "hint": "If a new foreign key between these entities was created in the database, try reloading the schema cache.",
+    "message": "Could not find a relationship between cities and countries in the schema cache"
+  }
+
+As you can see, PostgREST couldn't find the relationship in the schema cache. See the section :ref:`schema_reloading` to solve this issue.
+
+Stale Function Metadata
```

Comment:
[Signature](https://developer.mozilla.org/en-US/docs/Glossary/Signature/Function) might be a more precise term.
```suggestion
Stale Function Signature
```